### DISCUSSION TITLE
test documentation

### DISCUSSION THREAD

**Discussion by dsgipe** [graphql-java-kickstart/graphql-spring-boot#408] [Q&A] [answered]
Trying to get graphql-spring-boot-starter-test up and working and I am struggling to find appropriate solutions. Digging around on github and stack overflow hasn't resulted in any success. Can you point me to documentation on the graphql-spring-boot-starter-test, or provide a demo project that shows how to use the tool?

Thanks

**Comment by oliemansm** [comment, marked as answer]
Documentation needs to be written, but that might take a while. You can take a look at the test cases here for example usage: https://github.com/graphql-java-kickstart/graphql-spring-boot/blob/master/example-graphql-tools/src/test/java/com/graphql/sample/boot/GraphQLToolsSampleApplicationTest.java.
